subsystem/31.0.0/pom.xml

```diff
@@ -31,22 +31,10 @@
 
     <name>WildFly: Legacy Test 31.0.0 Subsystem Controller</name>
     <description>The legacy Test 31.0.0 subsystem controller</description>
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <configuration>
-                    <source>10</source>
-                    <target>10</target>
-                </configuration>
-            </plugin>
-        </plugins>
-    </build>
```

Comment:
Just FYI @kabir , removing this, it seems my IDE added this plugin configuration during the development time